src/runtime/SandBox.h

```diff
@@ -42,6 +42,9 @@ class SandBox : public gc {
         bool isConstructor;
         bool isAssociatedWithJavaScriptCode;
         bool isEval;
+#ifdef ESCARGOT_DEBUGGER
+        uint32_t executionStateDepth;
+#endif /* ESCARGOT_DEBUGGER */
```

Comment:
I would put these before the bools. This usually reduces the struct size since less alignment gap is needed.